cmd/kube-aggregator/pkg/apis/apiregistration/v1alpha1/types.generated.go

```diff
@@ -1244,7 +1242,13 @@ func (x *APIService) CodecEncodeSelf(e *codec1978.Encoder) {
 				z.EncSendContainerState(codecSelfer_containerArrayElem1234)
 				if yyq94[2] {
 					yy102 := &x.ObjectMeta
-					yy102.CodecEncodeSelf(e)
+					yym103 := z.EncBinary()
+					_ = yym103
+					if false {
+					} else if z.HasExtensions() && z.EncExt(yy102) {
+					} else {
+						z.EncFallback(yy102)
```

Comment:
We have to create ugorji codecs for meta/v1 now that we have real structs there.